apps/src/tests/TestPR3045.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { NavigationContainer, ParamListBase } from '@react-navigation/native';
+import { NativeStackNavigationProp, createNativeStackNavigator } from '@react-navigation/native-stack';
+import { Button, ScrollView } from 'react-native';
+import { Text } from 'react-native-gesture-handler';
```

Comment:
I don't think this should be imported from `gesture-handler`. I think you want to import this from `react-native`.